ui-common/src/components/EntryDetailed/EntryDetailed.tsx

```diff
@@ -136,18 +136,10 @@ export const EntryDetailed = () => {
     }, [focusedEntryId]);
 
     return <React.Fragment>
-      {isLoading && <div style={{textAlign: "center", width: "100%", marginTop: 50}}><img alt="spinner" src={spinner} style={{height: 60}}/></div>}
-      {!isLoading && entryData && <EntryTitle
-            protocol={entryData.protocol}
-            data={entryData.data}
-            elapsedTime={entryData.data.elapsedTime}
-        />}
-        {!isLoading && entryData && <EntrySummary entry={entryData.base} namespace={entryData.data.namespace} />}
-        <React.Fragment>
-            {!isLoading && entryData && <EntryViewer
-                representation={entryData.representation}
-                color={entryData.protocol.backgroundColor}
-            />}
-        </React.Fragment>
+        <LoadingWrapper isLoading={isLoading} loaderMargin={50} loaderHeight={60}>
+            {entryData && <EntryTitle protocol={entryData.protocol} data={entryData.data} elapsedTime={entryData.data.elapsedTime} />}
```

Comment:
maybe this will be nicer:
{entryData && <React.Fragment> 
<EntryTitle../>
<EntrySummary../>
<EntryViewer../> 
</React.Fragment>}

Reply:
yesss